sample/src/extension.ts

```diff
@@ -184,6 +195,33 @@ ${stderr}`);
             vscode.window.showErrorMessage((error as Error).toString());
         }
     });
+
+    const sampleSDKlistVersions = vscode.commands.registerCommand('sample.dotnet-sdk.listVersions', async (getRuntimes : boolean) => {
+        if (!getRuntimes) {
+            getRuntimes = JSON.parse(await vscode.window.showInputBox({
+                placeHolder: 'false',
+                value: 'false',
+                prompt: 'Acquire Runtimes? Use `true` if so, else, give `false`.',
```

Comment:
This looks like a sample, so maybe the answer should be 'no', but should this be localized?

Reply:
It'd be good to have localization for everything in this repo, but currrently most of these strings aren't